docs/monitoring/metrics.md

```diff
@@ -376,6 +376,7 @@ Parameters:
 - `dmax` - hard limit for how long an old metric should be retained
 - `ttl` - time-to-live for transmitted UDP packets
 - `addressingMode` - UDP addressing mode to use (UNICAST/MULTICAST)
+- `maxComponentLength` - limits the size of each scope component
```

Comment:
"length of each scope component" would be better, I don't think we us "name of scope component" anywhere in the docs.